packages/suite/src/views/wallet/staking/components/Redelegate/index.tsx

```diff
@@ -58,7 +61,7 @@ const Redelegate = () => {
                 </Text>
 
                 <Actions>
-                    {deviceAvailable.status && delegatingAvailable.status ? (
+                    {deviceAvailable.status && delegatingAvailable.status && !isFetchError ? (
```

Comment:
what would happen if this condition `{deviceAvailable.status && delegatingAvailable.status && !isFetchError` would be replaced with `!reasonMessageId`
- would it be easier to reade?
- would it mean the same?

Reply:
Ahoj Martin 👩‍❤️‍👩

squashed!

- yes
- no

This fetchError is from the different source.